src/lib/components/layout/speakers.svelte

```diff
@@ -8,23 +8,31 @@
   export let speakers: Speaker[];
 </script>
 
-<div class="mt-xx-large md:mt-xxx-large" id="speakers">
+<div
+  class="mt-xx-large md:mt-xxx-large md:w-11/12 max-w-6xl mx-auto"
+  id="speakers"
+>
   <div class="flex flex-col gap-14 items-center md:items-start">
-    <h1
-      class="flex text-7xl md:text-8xl font-normal md:w-11/12 max-w-6xl mx-auto"
-    >
-      Speakers
-    </h1>
+    <div class="space-y-8">
+      <h1 class="flex text-7xl md:text-8xl font-normal">Speakers</h1>
+
+      <p>
+        To view the full agenda, check out our <a href="#">🗓️ Event Schedule</a>
```

Comment:
@chrifro we need a CTA URL for this

Reply:
Here it is https://www.notion.so/gitpod/CDE-Universe-2023-Agenda-e410e877d7f745af9934a2a0e90f4467